client/src/components/zoomWarning.module.scss.d.ts

```diff
@@ -0,0 +1,13 @@
+declare namespace ZoomWarningModuleScssNamespace {
+    export interface IZoomWarningModuleScss {
+        zoomWarning: string;
+    }
+  }
+
+declare const ZoomWarningModuleScssModule: ZoomWarningModuleScssNamespace.IZoomWarningModuleScss & {
+    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
+    locals: ZoomWarningModuleScssNamespace.IZoomWarningModuleScss;
+  };
+
+export = ZoomWarningModuleScssModule;
+
```

Comment:
Okay this is interesting. Wanna explain how this is working? Never seen an interface between typescript and scss.

Reply:
Yeah this is from `gatsby-plugin-scss-typescript` initially - we abandoned this in favor of `gatsby-plugin-sass` due to some bugs we came across, but one beneficial thing it does is generates these type files for us - so I temporarily installed the plugin, ran `gatsby develop` to generate the interfaces, then deleted the plugin again. We should work through these problems at some point to make it easier to make modules, but for now you can also optionally copy an existing `.d.ts` file